patches/MCLOUD-5771__change_redis_cache_adapter__2.3.x.patch

```diff
@@ -0,0 +1,24 @@
+diff -Naur a/setup/src/Magento/Setup/Model/ConfigOptionsList/Cache.php b/setup/src/Magento/Setup/Model/ConfigOptionsList/Cache.php
+--- a/setup/src/Magento/Setup/Model/ConfigOptionsList/Cache.php
++++ b/setup/src/Magento/Setup/Model/ConfigOptionsList/Cache.php
+@@ -20,7 +20,7 @@ use Magento\Setup\Validator\RedisConnectionValidator;
+ class Cache implements ConfigOptionsListInterface
+ {
+     const INPUT_VALUE_CACHE_REDIS = 'redis';
+-    const CONFIG_VALUE_CACHE_REDIS = 'Cm_Cache_Backend_Redis';
++    const CONFIG_VALUE_CACHE_REDIS = '\\Magento\\Framework\\Cache\\Backend\Redis';
```

Comment:
Missing a `\` before `Redis`

Reply:
In such way it was delivered to the mainline. I prefer to leave it as is for compatibility with future releases and possible patches based on it.
https://github.com/magento/magento2/blob/2.4-develop/setup/src/Magento/Setup/Model/ConfigOptionsList/Cache.php

Reply:
Got'cha. It's obviously fine either way, I was just opting for consistency. Since that's the original version you're right to leave it as it is.